### ISSUE TITLE
v1.34.0 and later broke npm:node-fetch

### ISSUE THREAD

**Issue by jeremymeng** [denoland/deno#19413]
npm:node-fetch is broken since v1.34.0 and later

```ts
import fetch1 from "npm:node-fetch";
const url = "https://storageyumeng.blob.core.windows.net/%24root?restype=container";
const init =  {
    method: "PUT",
  }

const resp = await fetch1(url, init);
console.log(resp);
```

HTTP proxy shows that the request sent out now has extra `?resttype=container`, i.e., the query is now `container?restype=container` instead of just `container`

 storageyumeng.blob.core.windows.net /%24root?restype=container?restype=container

**Comment by bartlomieju** [comment]
Thanks for the reports. This has been fixed in https://github.com/denoland/deno/pull/19340 and will be released in v1.34.2, most likely tomorrow.

**Comment by bartlomieju** [comment]
Gonna close this one as v1.34.2 is en route.
